sdfObject/sdfobject-genericdefaulttransitiontime.sdf.json

```diff
@@ -22,27 +22,29 @@
       "sdfAction": {
```

Comment:
These look a lot like properties to me.

Reply:
I made a standalone sdfData definition to make these reusable as properties of an object type sdfProperty. We can compare to the generic level control, where the data structure has 2 more fields added. After we merge?

Reply:
This is certainly not incorrect SDF, but it strikes me as an abuse of sdfAction to just server as getter/setter pairs -- this is what properties were meant for.